stylesheet.css

```
@@ -0,0 +1,57 @@
+.header {
+    float: right;
+    margin: 25px;
+}
+    
+.login-btn {
+    background-color: #022432;
+    height: 60px;
+    width: 120px;
+    color: white;
+    font-weight: bold;
+    font-size: 12px;
+    border-radius: 5px;
+    text-decoration: none;
+    padding: 10px 30px;
+    margin-right: 15px;
+}
+    
+.mlogin-btn {
+    background-color: #022432;
+    height: 20px;
+    width:60px;
+    color: white;
+    font-weight: bold;
+    font-size: 12px;
+    border-radius: 5px;
+    text-decoration: none;
+    padding: 10px 20px;
+}
+    
+.main-top {
+    display: flex;
+    position:relative;
+}
+    
+.top-left {
+    background-color: #e6e6fa;
+    width: 50%;
+    height: 900px;
+    text-align: center;
+}
+    
+.top-right {
+    background-color: #d8bfd8;
+    width: 50%;
+    height: 900px;
+    text-align: center;
+    flex-direction: column;
+}
+
+.login-btn:hover {
+    opacity: 0.7;
+}
+
+.mlogin-btn:hover {
+    opacity: 0.7;
```

同じ指定なので&多分サイト全体でそうなると思うので共通のクラス指定が必要ですね

---

私も先生と同じ意見です。
ボタンエレメントにいくつか共通するcss要素があると思うので、

```suggestion
// class="button" みたいな使いまわせるクラスを作る
.button {
  opacity: 0.7;
  background-color: #022432;
 height: 20px;
  width: 60px;
  color: white;
  font-weight: bold;
  font-size: 12px;
  border-radius: 5px;
  text-decoration: none;
}
```
みたいな感じで両方のボタンエレメントに使う方がすっきりするかと思います

---

そうです。ただもっと言うと、common.cssみたいなのを用意してそこで管理するのと、hoverしたら0.7にするとかはサイト全体のことなのでさらに切り分けて

.button:hover {
  opacity: 0.7;
}
のように一個さらに切り出すがいいと思いました
逆にwidth: 60pxやheightもそこに含めると可変にしたい時に不便になるので、そこはまた切り出すがいいと思っています